lib/travis/vcs/git/clone.rb

```diff
@@ -145,7 +145,8 @@ def pull_args
           end
 
           def fetch_args
-            args = ""
+            args = " "
```

Comment:
is this expected change or is this accidental?

Reply:
yes, by purpose, in next previously it was " --quiet", now the space is needed somewhere before depth_flag, so either depth_flag needs to be changed, or space put here or in next line

Reply:
Thanks @GbArc for explaining, I was just making sure.